src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.scss

```diff
@@ -1,9 +1,18 @@
+.grafana-container {
+  margin-bottom: 25px;
+  overflow: hidden;
+  padding-top: 56.25%;
+  position: relative;
+}
+
 .grafana {
-  height: 600px;
+  height: 100%;
+  left: 0;
+  position: absolute;
+  top: 0;
   width: 100%;
-  z-index: 0;
 }
-
+/*
```

Comment:
Is `grafana_one`, `grafana_two`, `grafana_three `and `grafana_four `no longer needed?
If so we can remove this commented styles, otherwise we should remove the comment.

Reply:
@tspmelo hi, they have been removed.